piksi_tools/console/console.py

```diff
@@ -190,6 +210,25 @@ class SwiftConsole(HasTraits):
   update_view = Instance(UpdateView)
   log_level_filter = Enum(list(SYSLOG_LEVELS.itervalues()))
 
+
+########## TRY TO IMPLEMENT FIX TYPE AND #SATS HERE #########
+  
+  mode = Str('') 
+  num_sats = Int(0)
+  port = Str('')
+
+  running = Bool(True)
+  logging_button=Button('Log')
+  #logging = Bool(False)
+
+  file_name = File
+
+  CSV_or_JSON = Bool
+
+  ##########################################################
+
```

Comment:
Make this comment block more descriptive.  What part of the view do these veriables control are apply to.  Also not need for all the ####, normal line comments are sufficient.